### ISSUE TITLE
Issue on Table 1d in the paper

### ISSUE THREAD

**Issue by SchweitzerGAO** [SihanXU/nepa#10]
I wonder what exactly you mean by *Freeze Embedding* since there is no further explanation in the paper (If I ignored this, please just point out)

<img width="960" height="521" alt="Image" src="https://github.com/user-attachments/assets/03d7ab7b-2547-402b-93dc-e2edd6d8548f" />
Does this mean you just fix all the parameters in the patch embedding layer to randomly initialized values during training? Then how can it reach 73.1% accuracy?

**Comment by SihanXU** [comment]
Oh sorry for the unclear wording. We explain this in the LayerScale part of Section 4.2. During pretraining, the embedding layer is not frozen. The “freeze embedding” setting refers to the fine-tuning stage.
